### PR TITLE
python310Packages.cinemagoer: init 2022.2.11

### PR DESCRIPTION
**Pull request by superherointj**
Upstream has renamed package from `imdbpy` to `cinemagoer`.

### CODE REVIEW

**Comment by PedroHLC** [comment]
Result of `nixpkgs-review pr 185709` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>hypnotix</li>
    <li>python310Packages.cinemagoer</li>
    <li>python39Packages.cinemagoer</li>
  </ul>
</details>

**Comment by superherointj** [comment]
Result of `nixpkgs-review pr 185709` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>hypnotix</li>
    <li>python310Packages.cinemagoer</li>
    <li>python39Packages.cinemagoer</li>
  </ul>
</details>

**Review by PedroHLC — APPROVED** [review]

File: pkgs/top-level/python-aliases.nix [hunk 1]
```diff
@@ -90,6 +90,7 @@ mapAliases ({
   hdlparse = throw "hdlparse has been removed, it was using setuptools 2to3 translation feature, which has been removed in setuptools 58"; # added 2022-01-18
   hyperkitty = throw "Please use pkgs.mailmanPackages.hyperkitty"; # added 2022-04-29
   IMAPClient = imapclient; # added 2021-10-28
+  imdbpy = throw "imdbpy has been renamed to cinemagoer"; # added 2022-08-08
```
Comment (dotlambda) [inline pkgs/top-level/python-aliases.nix:93]: Why not
```suggestion
  imdbpy = cinamagoer; # added 2022-08-08
```
?

Comment (PedroHLC) [reply pkgs/top-level/python-aliases.nix:93]: > Why not
> ?

Probably it's a good idea to keep the throw: If you consider that [there are 17 packages still depending on the deprecated name](https://libraries.io/pypi/IMDbPY/dependents) and that https://pypi.org/project/imdbpy is not capable of doing the redirect alone, this throw here may help people to consider pushing the updates upstream. Who knows how many end-users scripts are also using the outdated package...

Comment (dotlambda) [reply pkgs/top-level/python-aliases.nix:93]: > What do you want to be done?

Nothing, I just wondered about the reasoning.

Comment (superherointj) [reply pkgs/top-level/python-aliases.nix:93]: > Nothing, I just wondered about the reasoning.

Didn't seem necessary. (A combination of: No references. Upstream had moved on. Niche package. Warning was added still.)

**Pull request merged**